plugins/janus_videoroom.c

```diff
@@ -2487,11 +2490,20 @@ int janus_videoroom_init(janus_callbacks *callback, const char *config_path)
 	{
 		janus_config_category *config_general = janus_config_get_create(config, NULL, janus_config_type_category, "general");
 		/*CARBYNE-RF*/
-		janus_config_item *item_rtsp = janus_config_get(config, config_general, janus_config_type_item, "rtsp_url");
-		if (item_rtsp && item_rtsp->value)
+		janus_config_item *dynamic_url = janus_config_get(config, config_general, janus_config_type_item, "dynamic_url");
+		if (dynamic_url && !janus_is_true(dynamic_url->value) ){
```

Comment:
It seems Janus are doing { in a new line so please update here and other places you've added {} accordingly like Janus

Reply:
f (dynamic_url && !janus_is_true(dynamic_url->value) ) -> f (dynamic_url && !janus_is_true(dynamic_url->value))
Deletion of white spaces at the end of the if inside of it

Reply:
fixed